x/emissions/module/rewards/rewards_internal.go

```diff
@@ -333,9 +333,17 @@ func GetConsensusScore(
 		if consensusLosses[i].IsZero() {
 			consensusLosses[i] = epsilon
 		}
-		rLossOverConsensusLoss, err := rLoss.Quo(consensusLosses[i])
-		if err != nil {
-			return alloraMath.ZeroDec(), err
+		var rLossOverConsensusLoss alloraMath.Dec
+		if rLoss.Gt(consensusLosses[i]) {
+			rLossOverConsensusLoss, err = rLoss.Quo(consensusLosses[i])
+			if err != nil {
+				return alloraMath.ZeroDec(), err
+			}
+		} else {
```

Comment:
abs(log(1/2)) == log(2/1) is the principle here

Reply:
Like tyler said, this change fits with EQ32

Reply:
long rabbit hole later... plz see my latest changes